packages/contracts/contracts/token/Vesting.sol

```diff
@@ -14,8 +14,6 @@ import {DateTime} from "../libraries/DateTime.sol";
 import "hardhat/console.sol";
```

Comment:
We can probably remove this import, right? Or do we need to print something in this contract?

Reply:
True